snakemake/workflow.py

```diff
@@ -558,7 +558,7 @@ def files(items):
                         subworkflow.snakefile,
                         workdir=subworkflow.workdir,
                         targets=subworkflow_targets,
-                        configfiles=[subworkflow.configfile],
+                        configfiles=subworkflow.configfile,
```

Comment:
Configfiles has to be a list. 
```suggestion
                        configfiles=[subworkflow.configfile] if subworkflow.configfile else None,
```